### ISSUE TITLE
Is there a way to show only the days with events? 

### ISSUE THREAD

**Issue by rodrigoosrn** [Tibolte/AgendaCalendarView#67]
**Is there a way to show only the days with events?** 

I don't wanna show all days or the days between minDate and maxDate, but show only the days with events.
Is there a way?

**Comment by Xiaoning1943** [comment]
Yes, you can change the CalendarManager.loadEvents function, traverse your events and add them to CalendarManager.mEvents replace of traverse weekItem.
